plugins/inputs/file/README.md

```diff
@@ -1,16 +1,17 @@
 # File Input Plugin
 
-The file plugin updates a list of files every interval and parses the contents
-using the selected [input data format](/docs/DATA_FORMATS_INPUT.md).
+The file plugin parses the complete contents of a file each interval using the
+selected [input data format][].
 
-Files will always be read in their entirety, if you wish to tail/follow a file
-use the [tail input plugin](/plugins/inputs/tail) instead.
+**Note:** If you wish to parse only newly appended lines use the [tail][] input
```

Comment:
> only newly appended lines
Not just newly appended lines, but if you only want to parse the file _once_, you should use tail.

Reply:
said another way, if you don't want the file to be parsed from the beginning every interval, you should use tail